src/server/cluster/cluster_family.cc

```diff
@@ -289,11 +289,20 @@ void ClusterFamily::DflyCluster(CmdArgList args, ConnectionContext* cntx) {
     return DflyClusterGetSlotInfo(args, cntx);
   } else if (sub_cmd == "CONFIG") {
     return DflyClusterConfig(args, cntx);
+  } else if (sub_cmd == "MYID") {
+    return DflyClusterMyId(args, cntx);
```

Comment:
It looks like Redis implements `CLUSTER MYID`: https://redis.io/commands/cluster-myid/

As such, why do we want to implement `DFLYCLUSTER MYID` instead of being compatible with Redis (i.e. `CLUSTER MYID`)?

Reply:
As this command will be used for managing the cluster, and should not be used by server clients I prefer having it under dflycluster

Reply:
Gotcha, SG